src/less/icon/foreground/ds6/foreground.less

```diff
@@ -74,10 +74,6 @@ svg {
         .foreground-icon-image-mixin('mic');
     }
 
-    &.icon--overflow {
```

Comment:
Was this removed on purpose?

Reply:
I moved this style into `src/less/icon/foreground/base/foreground.less` since it's now shared between DS4 and DS6

Reply:
Aha! Of course. Thanks.